lib/pbench/server/database/alembic/versions/e695e86f722a_update_api_key.py

```diff
@@ -0,0 +1,29 @@
+"""Update api_key table with "id" as primary_key
+
+Revision ID: e695e86f722a
+Revises: 5679217a62bb
+Create Date: 2023-04-27 18:59:31.914683
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = "e695e86f722a"
+down_revision = "5679217a62bb"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.add_column(
+        "api_keys", sa.Column("id", sa.Integer(), autoincrement=True, nullable=False)
+    )
+    op.add_column("api_keys", sa.Column("name", sa.String(length=128), nullable=False))
+    op.create_unique_constraint(None, "api_keys", ["api_key"])
+
+
+def downgrade() -> None:
+    op.drop_constraint(None, "api_keys", type_="unique")
```

Comment:
> it might also revoke the primary key status?

Um..._can_ we have a table with **no** primary key?  (I didn't think such a thing was possible....)

> Now I want to poke around the upgraded DB with psql to see...

I would support that.  (Better you than me, anyway.)

Reply:
Ha. So, for what it's worth... I pulled the revision file and fired up a local container:

```
                                            Table "public.api_keys"
 Column  |            Type             | Collation | Nullable | Default | Storage  | Stats target | Description 
---------+-----------------------------+-----------+----------+---------+----------+--------------+-------------
 api_key | character varying(500)      |           | not null |         | extended |              | 
 created | timestamp without time zone |           | not null |         | plain    |              | 
 user_id | character varying           |           | not null |         | extended |              | 
 id      | integer                     |           | not null |         | plain    |              | 
 name    | character varying(128)      |           | not null |         | extended |              | 
Indexes:
    "api_keys_pkey" PRIMARY KEY, btree (api_key)
    "api_keys_api_key_key" UNIQUE CONSTRAINT, btree (api_key)
Foreign-key constraints:
    "api_keys_user_id_fkey" FOREIGN KEY (user_id) REFERENCES users(id)
Access method: heap
```

So `user_id` is the foreign key reference to `users.id`, which hasn't changed; but `api_keys` is still the primary key instead of `id`.

This unfortunately means that alembic isn't able to detect and implement the primary key change automatically. I'll have to figure out how to do this manually and post an update here.